src/api/PacbioRun.js

```diff
@@ -1,41 +1,19 @@
 import handlePromise from './PromiseHelper'
 
-const NO_OF_COLUMNS = 12
-const NO_OF_ROWS = 8
-
-const columns = () => {
-  return Array.from(Array(NO_OF_COLUMNS), (e, i) => String(i + 1))
-}
-
-const rows = () => {
-  return Array.from(Array(NO_OF_ROWS), (e, i) => String.fromCharCode(65 + i))
-}
-
-const buildWell = (row, column) => {
-  return  {
-    row: row, 
-    column: column, 
+// const buildWell = (row, column, generateHiFiDefault='', preExtenstionTimeDefault
```

Comment:
Apparently all of my comments about your comments were outdated but don't know if this comment is also outdated? :)